account_spread_cost_revenue/i18n/es.po

```diff
@@ -0,0 +1,804 @@
+msgid "Delete Move"
+msgstr "Borrar movimiento"
+
+#. module: account_spread_cost_revenue
+#: code:addons/account_spread_cost_revenue/models/account_spread_line.py:149
+#, python-format
+msgid "Deleted move %s"
+msgstr "Movimiento borrado %s"
+
+#. module: account_spread_cost_revenue
+#: model:ir.model.fields,field_description:account_spread_cost_revenue.field_account_spread_line_name
+msgid "Description"
+msgstr "Descripción"
+
+#. module: account_spread_cost_revenue
+#: model:ir.ui.view,arch_db:account_spread_cost_revenue.view_account_spread
+msgid "Details"
+msgstr "Detalles"
+
+#. module: account_spread_cost_revenue
+#: model:ir.model.fields,help:account_spread_cost_revenue.field_res_company_allow_spread_planning
+msgid "Disable this option if you do not want to allow the spreading before the invoice is validated."
+msgstr "Deshabilita esta opción si quieres permitir la distribución antes de la validación de la factura."
+
+#. module: account_spread_cost_revenue
+#: model:ir.model.fields,field_description:account_spread_cost_revenue.field_account_spread_display_create_all_moves
+msgid "Display Button All Moves"
+msgstr "Botón de visualización de todos los movimientos"
+
+#. module: account_spread_cost_revenue
+#: model:ir.model.fields,field_description:account_spread_cost_revenue.field_account_spread_display_move_line_auto_post
+msgid "Display Button Auto-post lines"
+msgstr "Botón de visualización de lineas auto enviadas"
+
+#. module: account_spread_cost_revenue
+#: model:ir.model.fields,field_description:account_spread_cost_revenue.field_account_spread_display_recompute_buttons
+msgid "Display Buttons Recompute"
+msgstr "Botones de recalculo"
+
+#. module: account_spread_cost_revenue
+#: model:ir.model.fields,field_description:account_spread_cost_revenue.field_account_spread_display_name
+#: model:ir.model.fields,field_description:account_spread_cost_revenue.field_account_spread_invoice_line_link_wizard_display_name
+#: model:ir.model.fields,field_description:account_spread_cost_revenue.field_account_spread_line_display_name
+#: model:ir.model.fields,field_description:account_spread_cost_revenue.field_account_spread_template_display_name
+msgid "Display Name"
+msgstr "Nombre mostrado"
+
+#. module: account_spread_cost_revenue
+#: model:ir.model.fields,help:account_spread_cost_revenue.field_res_company_auto_archive
+msgid "Enable this option if you want the cron job to automatically archive the spreads when all lines are posted."
+msgstr "Habilita esta opción si quieres una acción planificada que archive automáticamente las distribuciones cuando todas las lineas sean publicadas."
+
+#. module: account_spread_cost_revenue
+#: model:ir.model.fields,help:account_spread_cost_revenue.field_res_company_force_move_auto_post
+msgid "Enable this option if you want to post automatically the accounting moves of all the spreads."
+msgstr "Habilita esta opción si quieres publicar automáticamente los asientos contables de todas las distribuciones."
```

Comment:
`post automatically the accounting moves` se traduciría por `asentar automáticamente los asientos contables`.